### PR TITLE
Use hyphenating in commands pre-fixed with "no", such as changing from `nosound` to `no-sound`

### PR DESCRIPTION
**Pull request by pdelfino**
Hi guys,

This PR address issue #1660. As a *general instruction*, John asked me to insert the name of the files being changed in commit messages. So, I did it.

I did 1 commit for each file changed. Sometimes, one file had something like 2, 3 or 4 lines changed due to the renaming.

I also did 1 commit for each `magit-file-rename` which means 4 commits. 

I know how to squash but I was just unsure if I should squash these commits. It would be nice to receive suggestions if these renaming should be in master history or not.

Since these changes could cause problems, I tested `no-sound`, `no-image`, `no-script`, and `no-webgl`. 
I did not break anything. Everything is working :wink:

### CODE REVIEW

File: documents/CHANGELOG.org [hunk 1]
```diff
@@ -539,7 +539,7 @@ If possible!
 
   Help to implement additional backends is welcome!
 
-- New =nowebgl-mode=.
+- New =no-webgl-mode=.
```
Comment (Ambrevar) [inline documents/CHANGELOG.org:542]: Don't update the ChangeLog previous entries, since older entries refer to older code.

File: source/dom.lisp [hunk 2]
```diff
@@ -65,7 +65,7 @@
   img-element input-element ins-element kbd-element label-element legend-element
   (li-element list-element) link-element (main-element semantic-element) map-element
   (mark-element semantic-element) meta-element meter-element (nav-element semantic-element)
-  noscript-element object-element (ol-element list-element) optgroup-element
+  no-script-element object-element (ol-element list-element) optgroup-element
```
Comment (Ambrevar) [inline source/dom.lisp:68]: Not sure this is correct.  @aartaka ?

Comment (aartaka) [reply source/dom.lisp:68]: It's not! HTML is standardized, and we're borrowing from HTML here.

**Comment by pdelfino** [comment]
OK. The changes on CHANGE.log and `dom.lisp` were reverted and then I squashed them. I think it is good to go! :wink:

**Comment by jmercouris** [comment]
Thank you!

**Comment by Ambrevar** [comment]
Wait, this should not have been merged, it breaks backward compatibility!

I suggest to revert it, we dont' have a good way to deal with deprecated modes.  We can keep this for 3.0, it's a trivial, non-urgent change anyways.

**Comment by jmercouris** [comment]
Right, I forgot about that! I almost think it would be easier at this point to merge everything to the 2 series branch and work on that (except for this). What do you think?

**Comment by jmercouris** [comment]
I can also revert it of course, I let me see if I could turn it into 1 revert commit somehow

**Comment by Ambrevar** [comment]
> Right, I forgot about that! I almost think it would be easier at this point to
> merge everything to the 2 series branch and work on that (except for this). What
> do you think?

What do you mean with "everything" and "that"? :p

**Comment by Ambrevar** [comment]
For now I suggest we revert as I'd like to prepare 2.2.

**Comment by jmercouris** [comment]
> > Right, I forgot about that! I almost think it would be easier at this point to
> > merge everything to the 2 series branch and work on that (except for this). What
> > do you think?
> 
> What do you mean with "everything" and "that"? :p

It was clear in my head :-D. Reading it again, no idea... Ay yai yai. Anyways, I will revert.

**Pull request merged**